ui/imports/shared/panels/ConnectionWarnings.qml

```diff
@@ -15,6 +15,7 @@ Loader {
     property int connectionState: -1
     property var chainIdsDown: []
     property bool completelyDown: false
+    property int lastCheckedAtUnix: -1
```

Comment:
This should be a `double` everywhere, otherwise it might overflow

Reply:
replaced all instances in QML with `double`